projects/storefrontapp/server.ts

```diff
@@ -29,7 +29,7 @@ const ssrOptions: SsrOptimizationOptions = {
   logger: true,
 };
 
-const ngExpressEngine = NgExpressEngineDecorator.get(engine, ssrOptions);
+const cxExpressEngine = CxExpressEngineDecorator.get(engine, ssrOptions);
```

Comment:
[QUESTION]
Could you help me understand the reason behind changing the types to `cxExpressEngine`? This seems to require more migration effort since we have to change both the import path and the item name, whereas we could change only import paths.

Reply:
I thought it may be more efficient to rename it `cxExpressEngine` since it's no longer part of Angular code and we used to add `cx` prefix for some of our creations. I don't mind to revert this change.

Reply:
Related issue: https://github.com/SAP/spartacus/issues/18203

Reply:
Yes, let's revert it, please. The only reason we want to add our won NgExpressEngine i backward compatibility with the rest of our tools (e.g. OptimizedSsrEngine). Let's not broaden the amount of breaking changes to be migrated, unless really justified